core/src/test/java/it/pagopa/selfcare/external_api/core/ContractServiceImplTest.java

```diff
@@ -0,0 +1,50 @@
+package it.pagopa.selfcare.external_api.core;
+
+import it.pagopa.selfcare.external_api.api.FileStorageConnector;
+import it.pagopa.selfcare.external_api.api.MsCoreConnector;
+import it.pagopa.selfcare.external_api.model.documents.ResourceResponse;
+import it.pagopa.selfcare.external_api.model.token.Token;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import static it.pagopa.selfcare.commons.utils.TestUtils.mockInstance;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.*;
+
+
+@ExtendWith(MockitoExtension.class)
+class ContractServiceImplTest {
+
+    @InjectMocks
+    private ContractServiceImpl contractService;
+
+    @Mock
+    private MsCoreConnector msCoreConnectorMock;
+
+    @Mock
+    private FileStorageConnector storageConnectorMock;
+
+
+    @Test
+    void getContract(){
+        //given
+        String institutionId = "institutionId";
+        String productId = "productId";
+        Token token = mockInstance(new Token());
+        ResourceResponse response = mockInstance(new ResourceResponse());
```

Comment:
pay attention to mockInstance method, it uses reflection......look at https://ncrcoe.gitbooks.io/java-for-small-teams/content/specifics/1000_do_not_use_reflection.html